src/index.html

```diff
@@ -9,6 +9,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#home">
+        <img src="./images/logo.png" alt="moyo-logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="is-active nav__link" href="#apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops&computers"
+              data-qa="hover">
+                Laptops & Computers
```

Comment:
see [here](https://mateacademy.slack.com/archives/C050SHYFFR9/p1681912970291919) how we set indents while moving attributes to new line